llvm/include/llvm/DebugInfo/BTF/BTFParser.h

```diff
@@ -46,7 +46,7 @@ class BTFParser {
   // A copy of types table from the object file but using native byte
   // order. Should not be too big in practice, e.g. for ~250MiB vmlinux
   // image it is ~4MiB.
-  OwningArrayRef<uint8_t> TypesBuffer;
+  std::vector<uint8_t> TypesBuffer;
```

Comment:
also here

Reply:
From the comment, it seems like the expectation is that `TypesBuffer` will typically be much larger than any reasonable small buffer, so my assumption here is that `SmallVector`'s optimization would not help and just be a very minor overhead.